src/main/java/io/charkchalk/backend/entity/range/TimeRange.java

```diff
@@ -23,7 +26,7 @@ public class TimeRange {
     private Long id;
 
```

Comment:
Will need a `name` or not?

Reply:
In extreme cases, the course time is likely to conflict with the schedule; based on this consideration; there are currently no plans to name the `TimeRange`.